python/helper-image/launcher/launcher_test.go

```diff
@@ -305,3 +314,85 @@ func TestPathExists(t *testing.T) {
 		t.Error("pathExists failed on real path")
 	}
 }
+
+func TestHandlePydevModule(t *testing.T) {
+	tmp := os.TempDir()
+
+	tests := []struct {
+		description string
+		args        []string
+		shouldErr   bool
+		module      string
+		file        string
+		remaining   []string
+	}{
+		{
+			description: "plain file",
+			args:        []string{"app.py"},
+			file:        "app.py",
+		},
+		{
+			description: "-mmodule",
+			args:        []string{"-mmodule"},
```

Comment:
this is a valid arg?

Reply:
Yeah: Python implements standard unix arg parsing:

`python -m foo`
`python -mfoo` 

are the same